azurerm/resource_arm_securitycenter_subscription_pricing.go

```diff
@@ -0,0 +1,94 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/security/mgmt/2017-08-01-preview/security"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/suppress"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmSecurityCenterSubscriptionPricing() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmSecurityCenterSubscriptionPricingCreateUpdate,
+		Read:   resourceArmSecurityCenterSubscriptionPricingRead,
+		Update: resourceArmSecurityCenterSubscriptionPricingCreateUpdate,
+		Delete: resourceArmSecurityCenterSubscriptionPricingDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"tier": {
+				Type:             schema.TypeString,
+				Required:         true,
+				DiffSuppressFunc: suppress.CaseDifference,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(security.Free),
+					string(security.Standard),
+				}, false),
+			},
+		},
+	}
+}
+
+//NOTE: seems default is the only valid pricing name:
+//Code="InvalidInputJson" Message="Pricing name '360k Sponsored' is not allowed. Expected 'default' for this scope."
+func resourceArmSecurityCenterSubscriptionPricingCreateUpdate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Since there's only update API, shall we rename it to `resourceArmSecurityCenterSubscriptionPricingUpdate` for better understanding before dig into details?